android/MobiledgeXSDKDemo/README.md

```diff
@@ -5,94 +5,17 @@
-
-### Computer Vision Demos
-Select one of the **Detection** or **Recognition** activities from the main menu. You can control several options, like which stats are displayed by selecting the "gear" icon to access **Computer Vision Settings**. 
-
-Things to try:
-- You may switch between the front and rear camera by tapping the camera icon.
-- Tap the 3-dot menu, and select "Play Video" to process a canned video instead of images from the camera. This function is useful for unattended demos or benchmarking.
-- Go to settings and turn on "Show Latency Stats after session" to get a stats summary that can be copy/pasted for additional use.
-
-#### Face Detection
-The Face Detection activity provides a visual comparison of the latency offered by an Edge cloudlet vs. that of a server in the public cloud. The Edge cloudlet can be determined in a few ways, in this order of priority:
-1. The result of "Find Closest Cloudlet", if performed.
-1. Selected from within the Cloudlet Details page options menu. This also activates the "Override Edge cloudlet hostname" value in **Computer Vision Settings**.
-1. "Edge Server" entry in the **Computer Vision Settings**, if updated by the user.
-1. The default hostname from the provisioning data at http://opencv.facetraining.mobiledgex.net/cvprovisioning.json.
-
-Images from the camera are sent to a server that uses OpenCV to detect faces in the image. The server returns coordinates of any faces, and the app renders rectangles around them. 
-
-#### Face Recognition
-Face Recognition is similar, but instead of only detecting the presence of a face in an image, the name of the subject will also be returned if the person is part of the training data. To add your face to the training data, tap the 3-dot menu, and select "Training Mode." You must be signed in to your Google account via the main menu so that your name can be associated with the face images. "Guest Training Mode" is also available to add images of another person, whose name you must enter. You own your images and any guest images you add, and you can remove them at any time, by selecting the appropriate menu entry.
-
-Note that for Recognition, only a single face is supported at a time. If more than one face appears in the given image, only one of them will be detected and recognized.
-
-#### Pose Detection
-Pose Detection uses OpenPose on a GPU-enabled cloudlet to detect the pose of human bodies. Like the other activities, images from the camera are sent to the cloudlet for processing. Instead of rectangular coordinates, points representing the bones of the pose(s) are received and rendered.
-
-There is no Edge/Cloud comparison for this activity. A single cloudlet is used for image processing.
-
-Note that Pose Detection does not use the result of "Find Closest Cloudlet" because there are a limited number of GPU-enabled cloudlets available.
```

Comment:
Just curious why are you deleting this, this functionality is still in the app? 
Maybe a link to DevPortal if you think it is very descriptive.

Reply:
Vasanth created https://developers.mobiledgex.com/product/mobiledgex-demo from this README, and I have updated that doc with the new features, including Driving Route mode. At the bottom of this shortened README, there is a link to that doc after "For more details".